torch/csrc/profiler/standalone/execution_trace_observer.cpp

```diff
@@ -86,7 +86,7 @@ inline std::string getValueShape(
   if (val.isTensor()) {
     auto& tensor = val.toTensor();
     if (tensor.defined() &&
-        tensor.unsafeGetTensorImpl()->does_not_have_symbolic_sizes_strides()) {
+        !tensor.unsafeGetTensorImpl()->has_symbolic_sizes_strides()) {
```

Comment:
What is the difference between the two here? Should we delete the other and flip the flag for everything?

Reply:
yes, we should delete the old one. It is only used here, was introduced a few weeks ago